Process Process-5:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-9:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-14:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-4:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-8:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-6:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-2:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-3:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-10:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-11:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-7:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-13:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Process Process-12:
Traceback (most recent call last):
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "./experiments/CPSO-NUMPY/main.py", line 98, in run
  File "./experiments/CPSO-NUMPY/main.py", line 348, in optimize
ZeroDivisionError: integer division or modulo by zero
Traceback (most recent call last):
  File "./experiments/CPSO-NUMPY/main.py", line 470, in <module>
    process.start()
  File "/opt/rh/rh-python38/root/usr/lib64/python3.8/multiprocessing/process.py", line 115, in start
    assert self._popen is None, 'cannot start a process twice'
AssertionError: cannot start a process twice
